packages/babel-plugin-transform-destructuring-fork/src/index.js

```diff
@@ -0,0 +1,654 @@
+// This plugin is a copy of `@babel/plugin-transform-destructuring` with
+// optimisations for array destructuring of React Hooks:
+// https://github.com/babel/babel/blob/b6486a2/packages/babel-plugin-transform-destructuring/src/index.js
+/* eslint-disable */
+const { declare } = require('@babel/helper-plugin-utils');
+const { types: t } = require('@babel/core');
+
+const forcedLoose = ['useReducer', 'useState'].map(
```

Comment:
There are only 2 differences:

- this `forcedLoose` array
- its usage in `toArray` below